docs/special.rst

```diff
@@ -91,6 +91,11 @@ For example, you can create a "ragged" array of integers::
     >>> dset[0] = [1,2,3]
     >>> dset[1] = [1,2,3,4,5]
 
+But unfortunately, NumPy in particular has no real support for "ragged" array, so
+everything is workarounds. The `awkward-array <https://github.com/scikit-hep/awkward-array>`_
```

Comment:
I think this is some of my words from an issue? I was writing a bit casually; I might need to have another go at it.

I'm also not sure whether to point people to awkward-array. I thought it allowed reading and writing ragged arrays as HDF5 vlen data, but it actually has its own way of saving its arrays in an HDF5 file, and it's only useful if you use the same tool to read it.

Reply:
Yes, it's from https://github.com/h5py/h5py/issues/1548.
Then, I will  wait for your further suggestion.